GenderPayGap.WebUI/Views/Submit/EnterCalculations.cshtml

```diff
@@ -44,131 +44,131 @@
             </p>
             <p>For differences in rates of pay and bonuses, a positive <span aria-hidden="true" aria-label="percentage">%</span> indicates that men in your organisation receive a higher rate than women in your organisation.</p>
             <p>A negative <span aria-hidden="true" aria-label="percentage">%</span> indicates that men in your organisation receive a lower rate than women in your organisation.</p>
+

```

Comment:
**Note for whole file:**
The fields in the file have been re-ordered - but it makes the diff look like I've changed everything!